spec/factories.rb

```diff
@@ -126,5 +126,27 @@
     trait :stolen_bike do
       bike { FactoryBot.create(:stolen_bike) }
     end
+
+    factory :customer_contact_potentially_found_bike do
+      creator { FactoryBot.create(:user) }
+      bike { FactoryBot.create(:stolen_bike) }
+      title { "We may have found your bike" }
```

Comment:
Nit: Is it possible to generate the title, body and creator_email from the inputs (the bike and match)? I feel like it would be better if they were consistently generated in the same way.

Reply:
good call thanks — updated in a54f38b2